lib/games_manager.rb

```diff
@@ -250,6 +228,16 @@ def lowest_scoring_visitor
     average_min[0]
   end
 
+  def highest_scoring_home
+    averages = total_home_goals.merge(total_home_games) do |team_id, goals, games|
+      (goals/games.to_f).round(2)
+    end
+    average_max = averages.max_by do |team_id, average|
```

Comment:
Nice start, we can safely condense these last 4 lines into 1 line, to replace ln.235-238
'(averages.max_by {|team_id, average| average})[0]'